src/Workspaces/Remote/Core/ProjectSystem/ILanguageServerProjectSystemService.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Remote.ProjectSystem;
+
+internal interface ILanguageServerProjectSystemService
+{
+    Task<ILoadedProjectSystemProject> LoadProjectAsync(string projectFilePath, ImmutableDictionary<string, string> buildProperties, CancellationToken cancellationToken);
```

Comment:
It would be good to document what the `cancellationToken` represents here. That is, why it would be called, what happens when it is called, etc.